lib/rails_utils/version.rb

```diff
@@ -1,5 +1,5 @@
 # frozen_string_literal: true
 
 module RailsUtils
-  VERSION = "4.0.0"
+  VERSION = "5.0.0"
```

Comment:
What's new in this version bump 5.0.0 and 6.0.0?

See how to author originally bumps his up.  
https://github.com/winston/rails_utils/commit/0d642246942a85d4c7c4af0802e29b606e127801

Reply:
updated the CHANGELOG accordingly